packages/graphql-codegen-cli/src/presets.ts

```diff
@@ -1,14 +1,12 @@
 import { DetailedError, Types } from '@graphql-codegen/plugin-helpers';
-import { resolve } from 'path';
 
 export async function getPresetByName(
   name: string,
   loader: Types.PackageLoaderFn<{ preset?: Types.OutputPreset; default?: Types.OutputPreset }>
 ): Promise<Types.OutputPreset> {
   const possibleNames = [`@graphql-codegen/${name}`, `@graphql-codegen/${name}-preset`, name];
-  const possibleModules = possibleNames.concat(resolve(process.cwd(), name));
```

Comment:
The default loader loads relative to the user config and any custom loader should handle this themselves, though perhaps the loader should also be given the config file location